src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -35,5 +38,59 @@ public BlobFeed(string accountName, string accountKey, string containerName, str
             Log = loggingHelper;
             RelativePath = relativePath;
         }
+
+        public string FeedContainerUrl => AzureHelper.GetContainerRestUrl(AccountName, ContainerName);
+
+        public async Task<bool> CheckIfContainerExists()
```

Comment:
Do we have this code anywhere else? If so we might want to share source with it as well.

Reply:
Actually the "CreateAzureContainer" has a bool to avoid failing in case the container exist already so there is no need to check for that.